doc/src/tutorial/001_adding_lalrpop.md

```diff
@@ -78,4 +78,19 @@ create a submodule `foo::grammar`):
 lalrpop_mod!(grammar);
 ```
 
+## Attributes
+
+The code that LALRPOP generates sometimes falls afoul of code linters for
+cosmetic reasons which don't apply to macro generated code. Currently, there is
+not a blanket `#[automatically_derived]` because action code provided
+by the user may be worth linting on.
```

Comment:
```suggestion
```suggestion
not a blanket `#[automatically_derived]` attribute in the generated parser
because action code provided by the user may be worth linting on.
```

Feel free to tell me I'm being overly pedantic, but it feels clearer to spell things out more explicitly.
```